src/main/controllers/RootController.ts

```diff
@@ -0,0 +1,26 @@
+import {AuthedRequest} from '../types/AuthedRequest';
+import {Response} from 'express';
+import { PageData } from '../interfaces/PageData';
+
+export class RootController {
+  public get(req: AuthedRequest, res: Response, view: string, data?: PageData): void {
+    RootController.render(req, res, view, data);
+  }
+
+  public post(req: AuthedRequest, res: Response, view: string, data?: PageData): void {
+    RootController.render(req, res, view, data);
+  }
+
+  private static render(req: AuthedRequest, res: Response, view: string, data: PageData): void {
```

Comment:
Why do we need a static method here?

Reply:
I can't call the 'render' method through 'this' unless (if I remember correctly) I made it public. I think I might be able to use the autobind decorator to call this and it have the correct scope